### PR TITLE
rename usePlaybackState in docs

### PR DESCRIPTION
**Pull request by dmathewwws**

### CODE REVIEW

File: docs/Documentation.md [hunk 1]
```diff
@@ -550,7 +550,7 @@ const MyComponent = () => {
 };
 ```
 
-#### useTrackPlayerProgress
+#### usePlaybackState
```
Comment (benjipott) [inline docs/Documentation.md]: useProgress instead of usePlaybackState

Comment (Guichaguri) [reply docs/Documentation.md]: As @benjipott alreay pointed out, `useProgress` and `usePlaybackState` are completely different hooks.

Comment (dmathewwws) [reply docs/Documentation.md]: Whoops! Sorry, my bad.

File: docs/Documentation.md [hunk 2]
```diff
@@ -550,24 +550,24 @@ const MyComponent = () => {
...
 
 ```jsx
 import React from 'react';
 import { Text, View } from 'react-native';
-import { useTrackPlayerProgress } from 'react-native-track-player';
+import { useProgress } from 'react-native-track-player';
 
 const MyComponent = () => {
-  const { position, bufferedPosition, duration } = useTrackPlayerProgress()
+  const { position, bufferedPosition, duration } = useProgress()
```
Comment (benjipott) [inline docs/Documentation.md]: bufferedPosition in V2 is named  `position`

**Comment by dmathewwws** [comment]
Sorry about that! fixed.

File: docs/Documentation.md [hunk 3]
```diff
@@ -550,29 +550,29 @@ const MyComponent = () => {
...
-#### useTrackPlayerProgress
-A hook alternative to the [Progress Component](#progresscomponent).
+#### useProgress
+A hook alternative to the the deprecated Progress Component.
 
 | State            | Type     | Description                      |
 | ---------------- | -------- | -------------------------------- |
 | position         | `number` | The current position in seconds  |
-| bufferedPosition | `number` | The buffered position in seconds |
+| buffered.        | `number` | The buffered position in seconds |
```
Comment (ovr) [inline docs/Documentation.md]: Unneeded dot `.` 😸

**Comment by dmathewwws** [comment]
:) fixed

**Comment by Guichaguri** [comment]
This PR has been done in the dev branch, but the function was renamed in v2. Can you recreate this PR into the v2 branch instead?

**Comment by curiousdustin** [comment]
Applied these changes to v2: https://github.com/react-native-kit/react-native-track-player/commit/ee7a480fb46b7de21bf5e119cfcc0e318a74dc87

Thanks @dmathewwws! :)

**Pull request closed without merging**